homeassistant/components/flipr/__init__.py

```diff
@@ -75,14 +76,21 @@ async def _async_update_data(self):
 class FliprEntity(CoordinatorEntity):
     """Implements a common class elements representing the Flipr component."""
 
-    def __init__(self, coordinator, flipr_id, info_type):
+    def __init__(
+        self, coordinator: DataUpdateCoordinator, description: EntityDescription
+    ) -> None:
         """Initialize Flipr sensor."""
         super().__init__(coordinator)
-        self._attr_unique_id = f"{flipr_id}-{info_type}"
-        self._attr_device_info = {
-            "identifiers": {(DOMAIN, flipr_id)},
-            "name": NAME,
-            "manufacturer": MANUFACTURER,
-        }
-        self.info_type = info_type
-        self.flipr_id = flipr_id
+        self.entity_description = description
+        if coordinator.config_entry:
+            flipr_id = coordinator.config_entry.data[CONF_FLIPR_ID]
+            self._attr_unique_id = (
+                f"{flipr_id}-{description.key}-{coordinator.config_entry.entry_id}"
```

Comment:
This changes the unique ID, we should not do that.
`f"{flipr_id}-{description.key}"` seems to match what you had before?

Reply:
I corrected it.
Is there a way to assert this value in a unit test like in flipr/test_sensors.py : I did not find how to access this value of _attr_unique_id from the hass.states elements ?

Reply:
If you want to do that you need to get the object from the entity registry

Reply:
OK, I did it : added an assert to verify that unique_id is stable.